### PR TITLE
[14.0][FIX] contract: invoice message create

### PR DESCRIPTION
**Pull request by marcelsavegnago**
cc @rvalyi @rousseldenis @pedrobaeza

### CODE REVIEW

**Review by mileo — APPROVED** [review]

**Review by rvalyi — APPROVED** [review]
BTW this is a port of https://github.com/OCA/contract/pull/537 from 12.0

File: contract/models/contract.py [hunk 1]
```diff
@@ -565,17 +565,18 @@ def recurring_create_invoice(self):
...
         """
-        invoice = self._recurring_create_invoice()
-        if invoice:
-            self.message_post(
-                body=_(
-                    "Contract manually invoiced: "
-                    '<a href="#" data-oe-model="%s" data-oe-id="%s">Invoice'
-                    "</a>"
+        invoices = self._recurring_create_invoice()
+        if invoices:
```
Comment (pedrobaeza) [inline contract/models/contract.py]: No need of this `if`

Comment (marcelsavegnago) [reply contract/models/contract.py]: done

**Pull request merged**